conf/qubes-kickstart.cfg

```diff
@@ -26,4 +26,54 @@ repo --name=dom0-updates --baseurl=file:///tmp/qubes-installer/yum/dom0-updates/
 @fedora
 @debian
 @whonix
+# weaks dependencies
+-adobe-source-code-pro-fonts
+-compat-f32-dejavu-sans-fonts
+-compat-f32-dejavu-sans-mono-fonts
+-compat-f32-dejavu-serif-fonts
+-crypto-policies-scripts
+-deltarpm
+-dnfdaemon-selinux
+-fips-mode-setup
+-flac
+-fwupd-plugin-flashrom
+-fwupd-plugin-modem-manager
+-gcc-gdb-plugin
+-geoclue2
+-geolite2-city
+-geolite2-country
+-gnupg2-smime
+-gstreamer1-plugins-good-qt
+-lame
+-libsss_autofs
+-libsss_sudo
+-libxcrypt-compat
+-libyui-gtk
+-libyui-mga-gtk
+-libyui-mga-qt
+-libyui-qt
+-libyui-qt-graph
+-mkpasswd
+-ntfs-3g-system-compression
+-oddjob-mkhomedir
+-openssl-pkcs11
+-opus-tools
+-perl-IO-Compress
+-perl-IO-Socket-SSL
+-perl-Math-BigInt
+-perl-Mozilla-CA
+-pigz
```

Comment:
Personally I'm always installing it and use for compressing backups - it's much faster. But it isn't needed in default setup.